src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj

```diff
@@ -1,197 +1,98 @@
-    <IntermediateOutputPath>..\..\out\$(SolutionName)\obj\$(Platform)\$(Configuration)\</IntermediateOutputPath>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <OutputPath>..\..\BehaviorsSDK\bin\AnyCPU\Debug\</OutputPath>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DocumentationFile>..\..\out\BehaviorsSdk\bin\AnyCPU\Release\Microsoft.Xaml.Behaviors.xml</DocumentationFile>
-  </PropertyGroup>
-  <PropertyGroup>
-    <!-- This must come after the inclusion of the targets file -->
-    <PlatformTarget>AnyCPU</PlatformTarget>
-  </PropertyGroup>
-  <PropertyGroup>
     <PackageId>Microsoft.Xaml.Behaviors.Wpf</PackageId>
-    <PackageVersion>1.0.1</PackageVersion>
+    <Version>1.0.1</Version>
     <Title>Microsoft.Xaml.Behaviors.Wpf</Title>
     <Authors>Microsoft</Authors>
     <Description>Easily add interactivity to your apps using XAML Behaviors for WPF. Behaviors encapsulate reusable functionalities for elements that can be easily added to your XAML without the need for more imperative code.</Description>
+    <Company>Microsoft Corporation</Company>
+    <Product>Microsoft.Xaml.Behaviors</Product>
     <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=2038239</PackageLicenseUrl>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2026353</PackageProjectUrl>
     <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=708511</PackageIconUrl>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>Behavior Action Behaviors Actions Blend Managed C# Interaction Interactivity Interactions</PackageTags>
-    <PackageOutputPath>..\..\scripts</PackageOutputPath>
-    <NuspecFile>..\..\scripts\Microsoft.Xaml.Behaviors.Wpf.nuspec</NuspecFile>
+    <RepositoryType>git</RepositoryType>
+    <PackageProjectUrl>https://github.com/Microsoft/XamlBehaviorsWpf</PackageProjectUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/XamlBehaviorsWpf/master/LICENSE</PackageLicenseUrl>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);TimestampNugetPackage</GenerateNuspecDependsOn>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <OutputPath>..\..\BehaviorsSDK\bin\AnyCPU\$(Configuration)\</OutputPath>
```

Comment:
Why is this "fixed" to `AnyCPU`?

Reply:
Where do you see that? I'm not setting architecture as it's implied in csproj

Reply:
I would not set the output path explicitly. Bad things tend to happen, esp once you add the second target.

Reply:
I tend to agree with Morten on this one. Publish the symbols on symbolsource or MyGet and leave them out of the NuGet package. With SourceLink, the source code will be downloaded, so really I would prefer to leave the symbols out for now.

Reply:
Bad things? Works fine. It puts each tfm in a subfolder. What sort of bad things?

Reply:
I believe it'll break with multi-tfm. The logic in the SDK for computing those is hairy.